regression/goto-instrument/nondet_static/main.c

```diff
@@ -0,0 +1,8 @@
+const int g_A = 100;
```

Comment:
Minor nit, feel free to ignore, but now you have extended the tests, is it worth given them slightly better names? e.g. perhaps `no_nondet_static_array` and `no_nondet_static_multi_array` or something like that?